proposals/extensions.md

```diff
@@ -116,6 +109,251 @@ public implicit extension E for S
 }
 ```
 
+## Open issue: disambiguating between extension type method and classic extension method
+
+Should we disambiguate when signatures match?
+```csharp
+var x = new C().M; // ambiguous
+new C().M(); // ambiguous once we mix all extensions
+
+class C { }
+
+implicit extension E for C
+{
+    public static void M() { }
+}
+
+static class Extensions
+{
+    public static void M(this C c) { }
+}
+```
+
+## Open issue: removing zero-arity-matches-any from var scenario
+
+By the current rules, there are some unfortunate interactions between member access and natural function type.
+> Note: When the result of such a member lookup is a method group and K is zero,
```

Comment:
It is not obvious what is the context for the quote (where it is taken from) and how it is relevant to the issue.